### DISCUSSION TITLE
Precall Combine on large fields?

### DISCUSSION THREAD

**Discussion by coatsy35** [Courseplay/Courseplay_FS22#2676] [Q&A]
After getting fed up with AD not fixing it's combine mode, I've started to play more and more using CP for unloading which for the most part is working well. However, on large fields the combine does not precall the trailer until around 75%. Is there a way to achieve this? as otherwise the combine fillip and then stops when it's nearly full as it takes a long time for the trailer to get there.

**Comment by Tensuko** [comment]
Fixed percent and won't be changed.

**Reply by coatsy35** [reply]
Why not make it a configurable option? why has it got to be fixed, what will it break?

**Reply by Tensuko** [reply]
The more user can adjust things, the more problems come up.
It will stay like this in our controlled prediction system and thats it.

**Comment by coatsy35** [comment]
But it doesn’t work properly! The paths are all over the place, often ride outside of the field and the trailers are never close enough to the combine in large fields such as the US maps and the oaks. Either we need to be able to alter settings or it needs fixing. AD paths work superb for unloading just they haven’t fixed the issue with on the move or when a combine is in the slot, so obviously trailer path and positioning can be better

**Reply by Tensuko** [reply]
There won't be any fix as there is nothing broken!
I tell you I use our mode many many times and it all works just fine.
We keep it like that and also other ppl report no problem with this.

**Reply by coatsy35** [reply]
So why do the trailers not make it to the combine in time in a large field? They just sit at the far end of the field until the combine gets to about 74% and then calls them, it’s not long enough for a trailer to make it to the other end of the field before the combine stops. The combine shouldn’t be stopped. The trailers start a path, then stop one, then start one again a lot of the time, trying to get to the combine when they are called. I will record this and load logs.

**Reply by Tensuko** [reply]
We will not change it!
There are als pathfinder calculations that needs to be done before the driver starts, on large fields it can take a while.

But again, no changes will be made to this!
